index.html

```diff
@@ -52,6 +52,146 @@ <h2>I'm a Software Developer</h2>
                 </ul>
             </div>
         </div>
+        <div class="works-section">
+            <div id="works-section-title">
+                <h1>My Recent Works</h1>
+            </div>
+            <div id="separator"></div>
+            <div id="featured-project">
+                <img id="featured-project-image" alt="Featured project image" src="src/featured-project-image.png">
+                <h2 id="featured-project-title">Multi-Post Stories</h2>
+                <p id="featured-project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="featured-project-languages">
+                    <li id="lang1">css</li>
+                    <li id="lang2">html</li>
+                    <li id="lang3">bootstrap</li>
+                    <li id="lang4">Ruby</li>
+                </ul>
+                <button class="featured-project-button" type="button">See Project</button>
+            </div>
```

Comment:
- [ ] From the requirements of this project, the content of each project should be aligned with flexbox. Please let's try to make sure each project section, not just the main wrapper uses flex for that. This is a very useful tool for layouting.